beefy-gadget/src/lib.rs

```diff
@@ -231,13 +235,20 @@ where
 	}
 
 	fn handle_finality_notification(&mut self, notification: FinalityNotification<Block>) {
-		debug!(target: "beefy", "🥩 Finality notification: {:?}", notification);
+		debug!(target: "beefy", "Finality notification: {:?}", notification);
```

Comment:
I liked the icon, any reason for removing? :)

Reply:
I kept it only for "user-level" messages, i.e. info/warn.